connectors/connectors.sh

```diff
@@ -85,6 +101,16 @@ fi
 update_connector "bigquery" "$BIGQUERY_CONNECTOR_VERSION"
 update_connector "gcs" "$GCS_CONNECTOR_VERSION"
 
+if [[ $UPDATED_GCS_CONNECTOR != true ]]; then
+  echo "GCS connector wasn't updated - no need to restart any services"
+  exit 0
+fi
+
+# Restart YARN NodeManager service on worker nodes so they can pick up updated GCS connector
+if is_worker; then
+  systemctl kill -s KILL hadoop-yarn-nodemanager
```

Comment:
Why don't use `systemctl restart`?

Reply:
`systemctl restart` will restart process with TERM signal that will cause graceful shutdown during which NM will tries to unregister - I want to avoid this.